package.json

```diff
@@ -11,13 +11,20 @@
     "format": "prettier --config .prettierrc.json --write 'src/**/*.js'",
     "format:check": "prettier --config config/.prettierrc.json --debug-check --list-different 'src/**/*.js'",
     "lint": "eslint 'src/**/*.js'",
-    "lint:fix": "eslint --fix 'src/**/*.js'"
+    "lint:fix": "eslint --fix 'src/**/*.js'",
+    "parcel": "parcel index.html",
+    "parcel-build": "parcel build index.html"
```

Comment:
I usually wouldn't like to have multiple bundlers around since it increases complexity.  Let's try using parcel for this and see what we think.

Reply:
parcel, is zero-config, so should be ok, right 😄 😅